# Secretory and transcriptomic responses of mantle cells to low pH in the Pacific oyster (Crassostrea gigas) - Supplementary Data

This GitHub repository provides the supplementary data referenced in the publication cited below.

## How to use and cite these files 

All files are publicly available and can be used for further research or other applications. However, if you utilize these resources in your work, we kindly request that you cite our original publication.

**Secretory and transcriptomic responses of mantle cells to low pH in the Pacific oyster (Crassostrea gigas).** Nicolás Zúñiga-Soto, Ingrid Pinto-Borguero, Claudio Quevedo, Felipe Aguilera. *Frontiers in Marine Science* **10**:1156831 (2023). https://doi.org/10.3389/fmars.2023.1156831

**Abstract**

Since the Industrial Revolution, the concentration of atmospheric carbon dioxide (CO2) due to anthropogenic activities has increased at unprecedented rates. One-third of the atmospheric anthropogenic CO2 emissions are dissolved in the oceans affecting the chemical equilibrium of seawater, which in turn leads to a decrease in pH and carbonate ion (CO32-) concentration, a phenomenon known as ocean acidification (OA). This chemical disequilibrium can be detrimental to marine organisms (e.g., mollusks) that fabricate mineralized structures based on calcium carbonate (CaCO3). Most studies on the effect of reduced pH in seawater have been conducted on the early developmental stages of shell-building invertebrates, given less attention to how adult individuals face OA stress. Here, we evaluate histological, secretory, and transcriptional changes in the mantle of adult oysters (Crassostrea gigas) exposure to ambient (8.0 ± 0.2) and reduced (7.6 ± 0.2) pH during 20 days. Most histological observations did not show differences in terms of mantle cell morphology. However, Alcian Blue/PAS staining revealed significant differences in the number of Alcian Blue positive cells in the mantle edge, suggesting a decrease in the secretory activity in this morphogenetic zone. Transcriptomic analysis revealed 172 differentially expressed genes (DEGs) between mantle tissues from adult oysters kept in normal and reduced pH conditions. Almost 18% of the DEGs encode secreted proteins that are likely to be contributing to shell fabrication and patterning. 17 of 31 DEGs encoding secreted proteins correspond to oyster-specific genes, highlighting the fact that molluscan shell formation is underpinned by a rapidly evolving secretome. The GO analysis of DEGs encoding secreted proteins showed that they are involved in the cellular response to stimulus, response to stress, protein binding, and ion binding, suggesting these biological processes and molecular functions are altered by OA. This study demonstrates that histology and gene expression profiling can advance our understanding of the cellular and molecular mechanisms underlying adult oyster tolerance to low pH conditions.

## Author contact

- [Felipe Aguilera](mailto:faguilera@udec.cl) (corresponding author)

## How to download data files

The files in this repository are ready for use. Simply click on the file, and you will be directed to the GitHub webpage to save it to your device.

## Index of data file contents

[01-Supplementary-Figures-Sequences.docx](https://github.com/faguil/Tyrosinase-Evolution-Molluscs/blob/main/01-Supplementary-Figures-Sequences.docx) contains supplementary figures tables used in this study.

- **Supplementary Figure 1.** Toloudine blue-stained section through the mantle tissue of *C. gigas* kept under nornal pH (8.0 ± 0.2) condition. 
- **Figure S2.** Maximum Likelihood (ML) phylogenetic tree of molluscan tyrosinase proteins obtained by RAxMLGUI v1.3 under the PROTGAMMAWAG substitution model.
- **Figure S3.** Bayesian Inference (BI) phylogenetic tree of molluscan tyrosinase proteins obtained by MrBayes v3.2 under the WAG + G substitution model.
- **Figure S4.** Sequence alignments of hypothetical proteins that belong to the microsynteny block between *C. gigas* scaffold 867 and *P. fucata* scaffolds 13287, 1286 and 19072.
- **Figure S5.** Quantitative PCR expression profiles of tyrosinase genes across the mantle tissue in *P. maxima* and *P. margaritifera*.
- **Supplementary dataset file S1.** Tyrosinase protein sequences used in this study.
- **Supplementary dataset file S2.** List of *Pinctada margaritifera* reference gene sequences used for qPCR analysis.
